### ISSUE TITLE
Could not resolve imports/virtual on 0.10.0

### ISSUE THREAD

**Issue by daniandl** [userquin/vuetify-nuxt-module#186]
Sorry to put an issue right after you pushed the update, but I can't run my app anymore after going from 0.9.1 to 0.10.0

```
✘ [ERROR] Could not resolve "virtual:vuetify-configuration"

    node_modules/vuetify-nuxt-module/dist/runtime/plugins/config.mjs:2:44:
      2 │ import { isDev, vuetifyConfiguration } from "virtual:vuetify-configuration";
        ╵                                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "virtual:vuetify-configuration" as external to exclude it from the bundle,
  which will remove this error and leave the unresolved path in the bundle.

✘ [ERROR] Could not resolve "#imports"

    node_modules/vuetify-nuxt-module/dist/runtime/plugins/config.mjs:3:27:
      3 │ import { useNuxtApp } from "#imports";
        ╵                            ~~~~~~~~~~

  You can mark the path "#imports" as external to exclude it from the bundle, which will remove this
  error and leave the unresolved path in the bundle.
```

The page does not render properly and theres 404 warnings of sass files on each reload.

**Comment by userquin** [comment]
upps, I didn't change the package exports, I'm going to check the playgroung in SB

**Comment by userquin** [comment]
@daniandl  it is working, you need to updated Nuxt to 3.9.0+, check the SB reproduction in the docs on in the readme file.

**Comment by userquin** [comment]
installing in real app and failing, I'll try to fix ASAP.

**Comment by userquin** [comment]
fixed in `v0.10.1`

**Comment by userquin** [comment]
@daniandl I need to review the new approach, my tests working but not sure if the new plugin template can be statically analysable.

I'm working on a new approach...
